ui/articles-aem-renderer/build.gradle

```diff
@@ -0,0 +1,44 @@
+apply plugin: 'com.jakewharton.butterknife'
+
+android {
+    defaultConfig {
+        vectorDrawables.useSupportLibrary = true
+
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+
+        javaCompileOptions {
+            annotationProcessorOptions {
+                arguments = [eventBusIndex: 'org.cru.godtools.tract.TractEventBusIndex']
+            }
+        }
+        multiDexEnabled true
```

Comment:
you shouldn't be enabling multi-dex for a library dependency, that should be defined at the application level.

Reply:
I was getting an error about it being only a single dex this is why I added it.

Reply:
hmm, in theory that should be handled at the app level only (at least based on my understanding of it)

Reply:
I'll do a little bit of research

Reply:
any library project added to the library or ui subdirectories gets this configuration automatically applied: https://github.com/CruGlobal/godtools-android/blob/develop/build.gradle#L108-L136

it saves having to duplicate a bunch of common configuration across all the modules